spec.md

```diff
@@ -33,7 +33,7 @@ effects on the host application.
 
 Starlark is intended to be simple. There are no user-defined types, no
 inheritance, no reflection, no exceptions, no explicit memory management.
-Execution is finite. The language does not allow recursion or unbounded loops.
+The language does not allow unbounded loops.
```

Comment:
I don't think its interesting to say anything about unbounded loops - once you have recursion, you have some form of unbounded loop. I'd just delete.